lib/cisco_node_utils/cmd_ref/vrf.yaml

```diff
@@ -64,8 +64,27 @@ route_distinguisher:
   _exclude: [ios_xr]
   get_value: '/^rd (\S+)$/'
   set_value: '<state> rd <rd>'
+  context: ['vrf context <vrf>']
```

Comment:
Ah - I see - this is not AF specific.

We really should have separate vrf.yaml and vrf_af.yaml to make the separate groupings clear and avoid having to repeat this all over the place.